src/main/java/com/maxmind/db/Reader.java

```diff
@@ -190,7 +198,61 @@ record = this.readNode(buffer, record, bit);
         return new DatabaseRecord<>(dataRecord, ipAddress, pl);
     }
 
-    private BufferHolder getBufferHolder() throws ClosedDatabaseException {
+    /**
+     * Creates a Networks iterator and skips aliased networks.
+     * Please note that a MaxMind DB may map IPv4 networks into several locations
+     * in an IPv6 database. This iterator will iterate over all of these locations
+     * separately. To disable skipAliasedNetworks, which iterates over the IPv4 networks 
```

Comment:
I still think it would be somewhat clearer if we replaced `skipAliasedNetworks` with `includeAliasesedNetworks`